Modix.Services.Test/AuthResultTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Modix.Common.Test;
+using Modix.Data.Models.Core;
+using Modix.Services.ErrorHandling;
+using NUnit.Framework;
+using Shouldly;
+
+namespace Modix.Services.Test
+{
+    public class AuthResultTests
+    {
+        [Test]
+        public void AuthResult_IsSuccess_IfUserHasClaims()
```

Comment:
TRIGGER WARNING: Me being picky.

The naming scheme we've been using so far is `UnitUnderTest_TestConditions_ExpectedResult`. So, for this one, IMO, it would be `Constructor_OwnedClaimsContainsRequiredClaims_IsSuccessIsTrue` with a name change from `hasClaims` to `ownedClaims` or `posessedClaims`.

Reply:
Renamed - though "IsSuccessIsTrue" seemed a bit redundant, so I just went with "IsSuccess".